server/lib/utils/slateUtils.ts

```diff
@@ -1,9 +1,16 @@
 import * as _ from "lodash";
 
-export const getAllInlinesAsArray = node => {
+export const getAllInlinesAsArray = nodeOrNodes => {
```

Comment:
I think some types would really help with things like this. Maybe not now though.